docs/medical-api/api-reference/patient/list-patients.mdx

```diff
@@ -12,8 +12,9 @@ api: "GET /medical/v1/patient"
 
 ## Response
 
-<ResponseField type="Patient[]">
-  All Patients receiving care at the specified Facility.
+All Patients receiving care at the specified Facility.
+
+<ResponseField name="patients" type="Patient[]">
```

Comment:
Need to update the `type="Patient[]"` here, and all the places where we list factilities/patients/docs

It's not returning `Patient[]`, but `{patients: Patient[]}`

My bad for not doing this properly to begin with, I was referencing the SDK returns when documenting, not the endpoints

Reply:
But this is saying there's a property with a field named `patients`, which is an array.

Do you want to have a wrapper around this to say its a JSON?

Reply:
Just noticed the `name="patients"` -> can ignore this!